packages/stargate/src/stargateclient.ts

```diff
@@ -136,19 +136,27 @@ export interface PrivateStargateClient {
   readonly tmClient: Tendermint34Client | undefined;
 }
 
+export interface StargateClientOptions {
```

Comment:
This symbol also needs to be added to `index.ts`

Reply:
Done :+1: